lib/minitest/reporters/base_reporter.rb

```diff
@@ -86,8 +86,12 @@ def print(*args)
       end
 
       def print_info(e, name=true)
-        print "#{e.exception.class.to_s}: " if name
-        e.message.each_line { |line| print_with_info_padding(line) }
+        if name
+          print "#{e.exception.class.to_s}: "
+          e.exception.message.each_line { |line| print_with_info_padding(line) }
+        else
+          e.message.each_line { |line| print_with_info_padding(line) }
```

Comment:
Perhaps it would be better to check for e class explicitly to avoid possible problems if someone will call the method with UnexpectedError and name == false.  Just to make the API easier to use.  What do you  think?